caffeineWiz.json

```diff
@@ -19,15 +19,19 @@
         "arm64",
         "arm"
     ],
-    "branch": "v0.1.0",
-    "license": "NeonAI Licensed",
-    "icon": "https://0000.us/klatchat/app/files/neon_images/icons/neon_paw.png",
-    "category": null,
+    "branch": "v0.1.1",
+    "license": "NeonAI License v1.0",
+    "icon": "https://raw.githubusercontent.com/NeonGeckoCom/CaffeineWiz.neon/master/logo.svg",
```

Comment:
sorry to be a pain... but master should be v0.1.1 to match the branch of the release

`https://raw.githubusercontent.com/NeonGeckoCom/CaffeineWiz.neon/master/logo.svg` -> `https://raw.githubusercontent.com/NeonGeckoCom/CaffeineWiz.neon/v0.1.1/logo.svg`

Reply:
Fixed, my mistake. I meant to specify v0.0.1